en/docs/learn/multi-channel-self-registration-and-account-confirmation.md

```diff
@@ -0,0 +1,404 @@
+            <li>By default this value is set to <code>EMAIL</code>.</li>
+        </ul>
+    </td>
+    </tr>
+    <tr>
+    <td>enable_resolve_notification_channel</td>
+    <td>
+        <ul>
+        <li>Enable server to determine whether to send notifications via a SMS or an email.</li> 
+        <li>Disabling this property would send  notifications via the channel defined by 
+        <code>default_notification_channel</code>.</li>
+        </ul> 
+    </td>
+    </tr>
+    <tr>
+    <td>skip_account_lock_on_verified_preferred_channel</td>
+    <td>
+        <ul>
+            <li>Allows a user to skip account confirmation if the user is already veirifed.</li>
+            <div class="admonition note">
+                <p class="admonition-title">Note</p>
+                <p>
+                    WSO2 Recommends to verify users after self registration than 
+                    pre verified user self registration. Therefore, it is recommended to set the property to 
+                    <code>false</code>. 
+                </p>
+            </div>
+        </ul>
+    </td>
+    </tr>
+    <tr>
+    <td>enable_detailed_api_response</td>
+    <td>
+        <ul>
+            <li>Enable detailed API response body.</li> 
+            <li>By default this value is set to <code>false</code>.</li>
+        <ul>
+    </td>
+    </tr>
+    </table>
+    
+    !!! tip
+            The **AccountConfirmation** and **SMSAccountConfirmation** templates are 
+            used to send this email notifications.
+        
+            You can edit and customize the email template. For more information, 
+            see [Customizing Automated Emails](../../learn/customizing-automated-emails).
+
+3.  Add an event publisher to `<IS_HOME/repository/deployment/server/eventpublishers`. For this 
+sample, `http output adapter` is used. Following is a sample publisher to call a REST Service 
+to send confirmation codes.
+
+    ??? info "Sample Event Publisher"
+        ```
+        <?xml version="1.0" encoding="UTF-8"?>
+        <eventPublisher name="HTTPOutputEventAdapter" processing="enable"
+            statistics="disable" trace="disable" xmlns="http://wso2.org/carbon/eventpublisher">
+            <from streamName="id_gov_sms_notify_stream" version="1.0.0"/>
+            <mapping customMapping="enable" type="json">
+                <inline>{"api_key"="4c9374",
+                    "api_secret"="FtqyPggE93",
+                    "from"="NEXMO",
+                    "to"={{mobile}},
+                    "text"={{body}}
+                    }</inline>
+            </mapping>
+            <to eventAdapterType="http">
+                <property name="http.client.method">httpPost</property>
+                <property name="http.url">https://rest.nexmo.com/sms/json</property>
+            </to>
+        </eventPublisher>
+        ``` 
+        
+        !!! note
+            This publisher users NEXMO as SMS REST service provider. For more information 
```

Comment:
This publisher **uses** NEXMO as **the** SMS REST service provider. **For more information on writing a custom http event publisher**, see.....

Reply:
Fixed: 030f591d9fd2591616bce502af05234321811549